util/module_logger.go

```diff
@@ -0,0 +1,260 @@
+package util
+
+import (
+	"fmt"
+	log "github.com/uber-common/bark"
+)
+
+// The level definition is stolen from logrus.
```

Comment:
That sucks. Let's do our own then, but perhaps we don't need to mention logrus, particular if we have added/changed the levels.